testing/tests/test_var_fps.py

```diff
@@ -69,15 +62,22 @@ def test_classification(self):
                 rend_path = rendition_dir + os.path.sep + filename
                 if not os.path.exists(rend_path):
                     continue
-                res = self.verify_file(src, rend_path)
+                np.random.seed(123)
+                random.seed(123)
+                start = timeit.default_timer()
+                verification_result = verifier.verify(src, [{'uri': rend_path}])
+                tamper = float(verification_result[0]["tamper"])
+                score_ul = float(verification_result[0]["tamper_ul"])
+                score_sl = float(verification_result[0]["tamper_sl"])
+                res = {'score': tamper, 'score_ul': score_ul, 'score_sl': score_sl}
                 res['master_filename'] = filename
                 res['rendition_type'] = rendition_name
                 res['is_correct'] = not tamper
                 results.append(res)
         df_res: pd.DataFrame = pd.DataFrame(results)
         df_res.set_index(['master_filename', 'rendition_type'], inplace=True)
         df_res.sort_index(inplace=True)
-        df_res['prediction'] = df_res['score'] > 0
+        df_res['prediction'] = df_res['score'] <= 0
```

Comment:
Wouldn't this condition for the prediction make Tamper the positive class when we've been treating the Correct class as the positive class?

Reply:
Tamper is a positive class (positive score). In this test, it was vice versa. Fixed.